.pipelines/e2e.yaml

```diff
@@ -58,6 +58,7 @@ jobs:
     displayName: Run AgentBaker E2E
     env:
       VHD_BUILD_ID: $(VHD_BUILD_ID)
+      LOCATION: $(LOCATION)
```

Comment:
What is the default? eastus?

Reply:
I set the default pipeline var value to westus3 since that's what our VHD build is currently set to, though abe2e's have been defaulting to eastus which has been causing problems since we haven't built anything there in a while

Reply:
I think cameron mentioned he set it to westus3